backEnd/prisma/schema.prisma

```diff
@@ -9,6 +9,12 @@ generator client {
   output   = "../generated/prisma"
 }
 
+model area {
+  id    Int    @id @default(autoincrement())
+  city  String @unique
```

Comment:
we should think about the edge case where the same city appears in different states, different countries, or even two instances of the same city name in the same country without states